CHANGELOG.md

```diff
@@ -1,5 +1,109 @@
+* Flow is now v0.104.0 ([59db059](https://github.com/facebook/react-native/commit/59db059) by [@mroch](https://github.com/mroch))
+* React is now at 16.9 ([40e8a5f](https://github.com/facebook/react-native/commit/40e8a5f), [0ccedf3](https://github.com/facebook/react-native/commit/0ccedf3) by [@TheSavior](https://github.com/TheSavior))
+* Use Metro for auto-collapsing internal stack frames ([77125a1](https://github.com/facebook/react-native/commit/77125a1) by [@motiz88](https://github.com/motiz88))
+* Move React error message formatting into ExceptionsManager ([2dadb9e](https://github.com/facebook/react-native/commit/2dadb9e) by [@motiz88](https://github.com/motiz88))
+* Improve VirtualizedList error message ([bef87b6](https://github.com/facebook/react-native/commit/bef87b6) by [@vonovak](https://github.com/vonovak))
+
+#### Android specific
+
+* Bump Hermes to v0.1.1 ([c21e36d](https://github.com/facebook/react-native/commit/c21e36d) by [@sunnylqm](https://github.com/sunnylqm))
+* Use centralized package for DoNotStrip annotation ([35fc0ad](https://github.com/facebook/react-native/commit/35fc0ad) by [@passy](https://github.com/passy))
+
+#### iOS specific
+
+* Do not override ActivityIndicator color when setting its size ([14b0ed4](https://github.com/facebook/react-native/commit/14b0ed4) by [@cabelitos](https://github.com/cabelitos))
+* fix display problems when image fails to load ([71d7d68](https://github.com/facebook/react-native/commit/71d7d68))
+* Renamed yoga podspec to Yoga ([8d02e17](https://github.com/facebook/react-native/commit/8d02e17) by [@axe-fb](https://github.com/axe-fb))
+* Update loading pre-bundled message ([eb92f81](https://github.com/facebook/react-native/commit/eb92f81) by [@rickhanlonii](https://github.com/rickhanlonii)) [TODO] Andorid and iOS progress bar
+
+### Deprecated
+
+* Deprecate method UIManagerModule.playTouchSound() ([e3ec8db](https://github.com/facebook/react-native/commit/e3ec8db) by [@mdvacca](https://github.com/mdvacca))
+* Deprecate UIManager.measureLayoutRelativeToParent ([e42009b](https://github.com/facebook/react-native/commit/e42009b) by [@mdvacca](https://github.com/mdvacca))
+* Move `TextInput`'s proptypes to `DeprecatedTextInputPropTypes` ([427b54e](https://github.com/facebook/react-native/commit/427b54e) by [@empyrical](https://github.com/empyrical))
+* Move `Text` component's proptypes to `DeprecatedPropTypes` ([0057cc9](https://github.com/facebook/react-native/commit/0057cc9) by [@empyrical](https://github.com/empyrical))
+
+#### Android specific
+
+* DrawerLayoutAndroid drawerPosition now expects a string, number is deprecated ([305b0a2](https://github.com/facebook/react-native/commit/305b0a2) by [@TheSavior](https://github.com/TheSavior))
+
+#### iOS specific
+
+* Deprecate RCTExport ([b771e20](https://github.com/facebook/react-native/commit/b771e20) by [@RSNara](https://github.com/RSNara))
```

Comment:
I think this is internal only as an implementation detail of unreleased TurboModule cc @RSNara